bin.src/make_job_document.py

```diff
@@ -37,9 +37,9 @@
                         help='Name of metrics package to load, defaults to validate_drp.')
     parser.add_argument('--spec', type=str, default="design",
                         help='Spec level to apply: minimum, design, or stretch')
-    parser.add_argument('--dataset_name', type=str, default="validation_data_hsc",
+    parser.add_argument('--dataset_name', type=str, default="gen3_rc2_subset",
```

Comment:
I think this should just be `rc2_subset`.  We removed the `gen3` part to try to reduce confusion in the world where there is no gen2